unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/RuntimeComponentFactory.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL.Components;
+using DCL.Controllers;
+using DCL.Helpers;
+using DCL.Models;
+using UnityEngine;
+
+namespace DCL
+{
+    public interface IRuntimeComponentFactory
+    {
+        IComponent CreateComponent(int classId, object model);
```

Comment:
I removing the `model` parameter in the subsequent PR, so for now I'll leave this code temporarily so it doesn't conflict with that change, it will be fixed later anyways.